Wikipedia/UI-V5/WMFHomeViewController.m

```diff
@@ -79,13 +78,21 @@ - (NSString*)title {
 
 #pragma mark - Accessors
 
+- (void)setSavedPages:(MWKSavedPageList* __nonnull)savedPages {
```

Comment:
Maybe that wasn't the case when I started the patch, I'd have to look at the commit when this was added for info, but I'll try removing it and seeing what happens :pray: